### ISSUE TITLE
bug remove_extra_spaces/remove_html_tags

### ISSUE THREAD

**Issue by legibe-com** [hakanbolat/phpStringCompare#1]
hi :-)

there's a little bug in the code, line 71:

```
if (!empty($params['remove_extra_spaces'])) {
	$this->_remove_extra_spaces = $params['remove_html_tags'];
}
```

should actually be:

```
if (!empty($params['remove_extra_spaces'])) {
	$this->_remove_extra_spaces = $params['remove_extra_spaces'];
}
```

also, there's two arrays ($lang1 and $lang2) containing a list of accents of non-english languages that is unused in the code, and also incomplete (i can think of "éèêàôö" in french, "ß" in german, "ñ" in spanish, ... the list must be somewhere on internet, google knows it for sure, but a function to get them all would be a more efficient approach, and stackoverflow will have that for sure).

it's a great script otherwise, thanks for sharing it, even though it was a long time ago! ( :

**Comment by hakanbolat** [comment]
Hi,
Thank you so much for your support and interest. I'll fix this bug.
As you mentioned it's written a long time ago. it's need some performances fixes and improvements :)

**Comment by legibe-com** [comment]
hi,

you're welcome!
yeah, people might still pass around here and look for such a script, it does a very good job so why not spending an hour to improve it a little!

thanks lots! :-)

**Comment by hakanbolat** [comment]
Fixed on [e4ea8e4](https://github.com/hakanbolat/phpStringCompare/commit/e4ea8e4f90b508bfee3f5367ae9d5f9f955eeb59)
